RecentActivity/src/org/sleuthkit/autopsy/recentactivity/Chromium.java

```diff
@@ -81,21 +81,27 @@
     private static final String WEBFORM_ADDRESS_QUERY_V8X = "SELECT first_name, middle_name, last_name, full_name, street_address, city, state, zipcode, country_code, number, email, date_modified, use_date, use_count" +
                                                             " FROM autofill_profiles, autofill_profile_names, autofill_profile_emails, autofill_profile_phones" +
                                                             " WHERE autofill_profiles.guid = autofill_profile_names.guid AND autofill_profiles.guid = autofill_profile_emails.guid AND autofill_profiles.guid = autofill_profile_phones.guid";
+    private static final String HISTORY_FILE_NAME = "History";
+    private static final String BOOKMARK_FILE_NAME = "Bookmarks";
+    private static final String COOKIE_FILE_NAME = "Cookies";
+    private static final String LOGIN_DATA_FILE_NAME = "Login Data";
+    private static final String WEB_DATA_FILE_NAME = "Web Data";
+
+    
     private final Logger logger = Logger.getLogger(this.getClass().getName());
     private Content dataSource;
     private IngestJobContext context;
 
     private static final Map<String, String> BROWSERS_MAP = ImmutableMap.<String, String>builder()  
-            .put(NbBundle.getMessage(Chromium.class, "Browser.name.Microsoft.Edge"), "Microsoft/Edge") 
-            .put(NbBundle.getMessage(Chromium.class, "Browser.name.Yandex"), "YandexBrowser") 
-            .put(NbBundle.getMessage(Chromium.class, "Browser.name.Opera"), "Opera Software") 
-            .put(NbBundle.getMessage(Chromium.class, "Browser.name.SalamWeb"), "SalamWeb") 
-            .put(NbBundle.getMessage(Chromium.class, "Browser.name.UC.Browser"), "UCBrowser")
-            .put(NbBundle.getMessage(Chromium.class, "Browser.name.Brave"), "BraveSoftware")
-            .put(NbBundle.getMessage(Chromium.class, "Browser.name.Google.Chrome"), "Chrome")
+            .put(NbBundle.getMessage(Chromium.class, "Browser.name.Microsoft.Edge"), "Microsoft/Edge/User Data/Default") 
+            .put(NbBundle.getMessage(Chromium.class, "Browser.name.Yandex"), "YandexBrowser/User Data/Default") 
+            .put(NbBundle.getMessage(Chromium.class, "Browser.name.Opera"), "Opera Software/Opera Stable") 
+            .put(NbBundle.getMessage(Chromium.class, "Browser.name.SalamWeb"), "SalamWeb/User Data/Default") 
+            .put(NbBundle.getMessage(Chromium.class, "Browser.name.UC.Browser"), "UCBrowser/User Data%/Default")
+            .put(NbBundle.getMessage(Chromium.class, "Browser.name.Brave"), "BraveSoftware/Brave-Browser/User Data/Default")
+            .put(NbBundle.getMessage(Chromium.class, "Browser.name.Google.Chrome"), "Chrome/User Data/Default")
```

Comment:
Just because I'm wondering - why are these bundle messages?

Reply:
Not sure why.  Removed them to just be straight text now.